src/ui/public/doc_table/doc_table.html

```diff
@@ -107,8 +107,15 @@
 </div>
 
 <div ng-if="hits != null && !hits.length" class="kbnDocTable__error">
-  <h2><i class="fa fa-meh-o"></i></h2>
-  <h4 i18n-id="common.ui.docTable.noResultsTitle"
-    i18n-default-message="No results found"
-  ></h4>
+  <div class="euiText euiText--extraSmall euiTextColor euiTextColor--subdued">
+    <icon type="'visualizeApp'" size="'m'" color="'subdued'" aria-hidden="true"></icon>
+
+    <br>
+    <br>
```

Comment:
What heathen method of spacing is this double br! 😄

You can use an `euiSpacer` here pretty easily with html if you need. Otherwise nitpick and at least make it <br/>

Reply:
Yeah, totally a hack because it's still angular. Do you mean using the `.euiSpacer` classes? Or is there a React directive for `EuiSpacer`?

Reply:
I used the classes